libs/design-system/src/lib/Components/LinkCard3Column/LinkCard3Column.stories.tsx

```diff
@@ -0,0 +1,203 @@
+      stroke="#69717E"
+      stroke-width="0.5"
+    />
+    <path
+      d="M4.82031 12.7314L1.11453 10.6025"
+      stroke="#69717E"
+      stroke-width="0.5"
+    />
+    <path
+      d="M4.87767 12.7761L18.9998 4.764L33.1218 12.7761V28.7929L18.9998 36.805L4.87767 28.7929V12.7761Z"
+      stroke="#69717E"
+      stroke-width="0.5"
+    />
+    <path
+      d="M1.25 10.6294L19 0.289326L36.75 10.6294V31.0326L19 41.2805L1.25 31.0326L1.25 10.6294Z"
+      stroke="#69717E"
+      stroke-width="0.5"
+    />
+  </svg>
+);
+
+const Icon2 = () => (
+  <svg
+    width="33"
+    height="37"
+    viewBox="0 0 33 37"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M1.02727 27.9036L16.7178 36.7266L31.6204 28.1226L32.8176 9.55968L16.8366 0.237935L0.944936 9.41299L1.02727 27.9036ZM9.52072 5.11211L1.78834 9.57641L7.95206 13.1717L15.7803 8.65211L9.52072 5.11211ZM10.0869 4.78524L16.8351 0.889131L22.9989 4.48445L16.3464 8.32524L10.0869 4.78524ZM16.9155 8.64706L23.5592 4.81129L31.9643 9.71399L25.4512 13.4743L16.9155 8.64706ZM25.7302 13.9636L32.2104 10.2222L31.5787 20.0171L25.4914 23.5316L25.7302 13.9636ZM24.9198 23.8617L25.1586 14.2936L17.1965 18.8905L17.0893 28.3331L17.133 28.3574L24.9198 23.8617ZM25.475 24.1915L31.5352 20.6926L31.0777 27.7855L25.3021 31.1201L25.475 24.1915ZM24.7304 31.4501L24.9033 24.5215L17.1384 29.0046L17.082 28.9733L17.0032 35.9114L24.7304 31.4501ZM24.885 13.8012L16.3493 8.97393L8.5124 13.4986L16.9175 18.4013L24.885 13.8012ZM8.30518 14.0297L16.6333 18.8875L16.5295 28.0224L8.34726 23.4801L8.30518 14.0297ZM8.35014 24.1259L16.5223 28.6625L16.4398 35.924L8.3825 31.3934L8.35014 24.1259ZM7.81785 31.0759L7.78551 23.8125L1.55689 20.3547L1.58903 27.5734L7.81785 31.0759ZM1.55401 19.7089L7.78263 23.1667L7.74048 13.7004L1.51108 10.0667L1.55401 19.7089Z"
+      fill="#69717E"
+    />
+  </svg>
+);
+
+const Icon3 = () => (
+  <svg
+    width="36"
+    height="41"
+    viewBox="0 0 36 41"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M4.53199e-07 9.64258L18 20.2497V40.4997L0 30.214L4.53199e-07 9.64258Z"
+      fill="#69717E"
+    />
+    <path
+      d="M35.5 29.9241L18.5 39.6384V20.25V19.9643L18.2538 19.8192L1.02007 9.66362L18 0.567228L35.5 9.94223V29.9241Z"
+      stroke="#69717E"
+    />
+  </svg>
+);
+
+export default {
+  component: LinkCard3Column,
+  title: 'Components/LinkCard3Column',
+} as Meta;
+
+const Template: Story<LinkCard3ColumnProps> = (args) => {
+  return (
+    <div style={{ backgroundColor: '#f1f1f1', padding: '14px' }}>
+      <LinkCard3Column {...args} />
+    </div>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  buttonText: 'Get Started',
+  buttonUrl: '#changeme',
+  description:
+    'Building on Flow is easy. Start building now with lorem ipsum et sigitus loranum prospitarius.',
+  title: 'Start Your Project',
+  tags: ['Tag', 'Lorem', 'Ipsum'],
+  items: [
+    {
```

Comment:
Can we add an interface for these?

Reply:
I think anything that will be passed as a prop should be an interface, as in the props interface to the component" ... although I'm uncertain when you should use `type` vs `interface` ... Either way, as long as we can statically check our props I'm happy.